### ISSUE TITLE
Free hosters support to be removed in the coming weeks!!

### ISSUE THREAD

**Issue by 123Venom** [123Venom/plugin.video.venom#79]
Sorry to say but I'm officially retiring free hoster support in Venom in the coming weeks.  I've maintained them for some time in OpenScrapers and they just became to much hassle when I do not use them myself at all, I just did them for you folks.  I stopped maintaining them a little over a month back because it seemed they were all slowly going behind Cloudflare's v2 challenge that we can not defeat.  I watched OpenScrapers dwindle down from 30 free hosters to about 12 was the last count I took.  They're just dying off folks and fewer developers wants to be bothered with them.  With Venom's new support for torrent pack files I feel they are just no longer needed.  I can slim down Venom substantially without them and there should be some speed benefits seen once all the code is removed.

**Comment by bjbca** [comment]
Cool, Real-Debrid and torrents work just fine, and while you are dropping the dependancy resolveurl it would be fine if you dropped script.module.metahandler also, i removed it from my private addon a while ago and everything has been working great with a trakt account.
  Awesome work @123Venom , have a great day ;)

**Comment by 123Venom** [comment]
`script.module.metahandler` is something to consider down the road...one thing at a time.  Right now I'm testing fully resolving links without resolveURL at all.  That dependency is being removed and the amount of code cleanup and removal has made the scrape process faster.  The sources module will be nearly half it's size once this transition is complete. I'll personally never use a free hoster again and was making myself ill testing and checking them...I just had enough of them.

**Comment by xyzfre** [comment]
I definitely agree about free hosters I’m not sure how people even still use them having a premium service is the best,, what does Metahandler do?

**Comment by 123Venom** [comment]
It's for handling of local watch indicators and used Only if you do not have a trakt account.  If you have a trakt account it's bypassed and does nothing so in that case I'd not worry about it.  It's been ages since I tested metadatahandler.  Perhaps @bjbca can explain what the issue with it was and why he removed it in his fork when he's using trakt..which it should not have done a thing in that case.

**Comment by bjbca** [comment]
In the beginning i found trakt unstable but its reliability became drastically improved, at least for me, so i started using it and its indicators, nice thing with trakt is it keeps track across all your devices. Metahandler could be unstable at times and if one didnt use the right version with the proper keys one would get a failure notice and it hasnt been properly updated in 2 years, so in good housekeeping i decided why carry extra baggage when you know you aren't going to use it... so i just removed it from import in the addon.xml and uninstalled it from kodi, I dont want extra clutter in kodi, it dosent get loaded, no checking for updates, one less thing for a repo to carry, freed up resources all the way around .... however small it all adds up.
